sparkbox/designsystemssurvey
# Add cachebust for fingerprinting

- creates new cachebust:build task for running the sass, js, and cachebust tasks consecutively
- only runs when NODE_ENV=production

To test locally:
- NOTE: in order to run this locally, you will need to be using Node 8.5 or higher (I set the app environment to 8.9.1, which is what I am using locally)
- run `npm install` to install the cachebust module
- run `NODE_ENV=production npm start`
You should see a hashed version of the css and js files show up in the `dist` directory and `index.html` should reference the hashed versions.

To test on Heroku:
- open the app and check to see that `index.html` is referencing the hashed versions of the JS and CSS files and check in the site source for the hashed versions of the files.
